tests/test_connector.py

```diff
@@ -56,7 +56,15 @@ def unix_server(loop, unix_sockname):
         runners.append(runner)
         await runner.setup()
         site = web.UnixSite(runner, unix_sockname)
-        await site.start()
+        try:
+            await site.start()
+        except OSError as exc:
+            if str(exc) == "AF_UNIX path too long":
+                if sys.platform == "linux":
+                    raise
+                pytest.skip("Some UNIXes has too strict limitation "
```

Comment:
Once this https://github.com/aio-libs/aiohttp/pull/3832 is merged, revert the hack.